AssortedGraphicUtils.cpp

```diff
@@ -582,3 +598,122 @@ QVector<ROMUtils::SaveData> AssortedGraphicUtils::CreateSaveData(AssortedGraphic
     }
     return result;
 }
+
+/// <summary>
+/// Cehck if the current input entry can be changed or deleted.
+/// </summary>
+/// <param name="entry">
+/// The struct data of the graphic entry.
+/// </param>
+/// <returns>
+/// return true if the entry is not used in any Tilesets or Rooms.
+/// </returns>
+bool AssortedGraphicUtils::CheckEditability(AssortedGraphicEntryItem &entry, unsigned int &find_l, unsigned int &find_r, unsigned int &find_t)
```

Comment:
What is the purpose of find_l, find_r, find_t?

Reply:
added the comment